### PR TITLE
Added procedure in #1416

### PR DESCRIPTION
**Pull request by j0rt3g4**
Added information #1416  about gMSA Accounts in the procedure.
After verifying functioning.
@JohanFreelancer9  this one needs to be CopyEdited. Thank you in advance!

### CODE REVIEW

File: Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md [hunk 1]
```diff
@@ -403,3 +403,76 @@ To configure your servers that are running Client Access services to stop using
    ```
 
 2. Although you don't have to do this immediately, you should eventually restart all client computers to clear the Kerberos ticket cache from the computer.
+
+
+## Configure using gMSA Accounts
+
+Now Let's try to add a gMSA account to manage the infrastructure
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md]:
```suggestion
You can now add a gMSA account to manage the infrastructure.
```

File: Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md [hunk 2]
```diff
@@ -403,3 +403,76 @@ To configure your servers that are running Client Access services to stop using
...
+
+### Create ASA Account
+
+Create gMSA account and allow servers in the AD-group "Exchange Servers" access to it. The account name will be "exchangeASA"
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md]:
```suggestion
Create a gMSA account and allow servers in the AD-group "Exchange Servers" access to it. The account name will be "exchangeASA".
```

File: Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md [hunk 3]
```diff
@@ -403,3 +403,76 @@ To configure your servers that are running Client Access services to stop using
...
+
+    ```Powershell
+    New-ADServiceAccount -DNSHostName "mail.contoso.com" -Name "exchangeASA" -KerberosEncryptionType AES128,AES256 -PrincipalsAllowedToRetrieveManagedPassword (Get-ADGroup -Identity "Exchange Servers") -SamAccountName "exchangeASA" -Description "Service account for Exchange servers to allow Kerberos authentication"
+    ```
+
+### Set gMSA account on servers
+
+    Create Credentials Objects with Blank password
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md]:
```suggestion
    Create credentials objects with blank password.
```

File: Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md [hunk 4]
```diff
@@ -403,3 +403,76 @@ To configure your servers that are running Client Access services to stop using
...
+
+    ```Powershell
+    $Credential = New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList "contoso\exchangeASA$",(new-object System.Security.SecureString)
+    ```
+    
+    Get CAS servers and clear old ASA credentials
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md]:
```suggestion
    Get CAS servers and clear old ASA credentials.
```

File: Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md [hunk 5]
```diff
@@ -403,3 +403,76 @@ To configure your servers that are running Client Access services to stop using
...
+  
+    ```Powershell
+    Get-ClientAccessService | Set-ClientAccessService -RemoveAlternateServiceAccountCredentials -CleanUpInvalidAlternateServiceAccountCredentials -Verbose
+    ```
+
+    Get CAS servers and add gMSA as ASA credential
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md]:
```suggestion
    Get CAS servers and add gMSA as ASA credential.
```

File: Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md [hunk 6]
```diff
@@ -403,3 +403,76 @@ To configure your servers that are running Client Access services to stop using
...
+    ```Powershell
+    Get-ClientAccessService | Set-ClientAccessService -AlternateServiceAccountCredential $Credential -Verbose
+    ```
+
+    Verify ASA credential deployment
+    ```Powershell
+    Get-ClientAccessService -IncludeAlternateServiceAccountCredentialStatus | Format-List Name, AlternateServiceAccountConfiguration
+    ```
+
+    Set internal Auth method to "Negotiate" as this supports both NTLM and Kerberos
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md]:
```suggestion
    Set internal Auth method to "Negotiate", as this supports both NTLM and Kerberos.
```

File: Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md [hunk 7]
```diff
@@ -403,3 +403,76 @@ To configure your servers that are running Client Access services to stop using
...
+    ```Powershell
+    Get-OutlookAnywhere | Set-OutlookAnywhere -InternalClientAuthenticationMethod Negotiate
+    ```
+
+     Pre-check if the SPN are in use (checks all AD-objects)
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md]:
```suggestion
     Pre-check if the SPN are in use (checks all AD-objects).
```

File: Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md [hunk 8]
```diff
@@ -403,3 +403,76 @@ To configure your servers that are running Client Access services to stop using
...
+    ```Powershell
+    ADObject -Filter '(servicePrincipalName -like "http/mail.contoso.com") -or (servicePrincipalName -like "http/autodiscover.contoso.com")' -Properties servicePrincipalName | Select-Object Name,servicePrincipalName
+    ```
+
+    Set SPN
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md]:
```suggestion
    Set SPN.
```

File: Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md [hunk 9]
```diff
@@ -403,3 +403,76 @@ To configure your servers that are running Client Access services to stop using
...
+    ```Powershell
+    Set-ADServiceAccount -Identity "exchangeASA" -ServicePrincipalNames "http/mail.contoso.com","http/autodiscover.contoso.com"
+    ```
+
+    Check afterwards that SPN are assigned
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md]:
```suggestion
    Check that the SPN is assigned.
```

File: Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md [hunk 10]
```diff
@@ -403,3 +403,76 @@ To configure your servers that are running Client Access services to stop using
...
+    ```Powershell
+    Get-ADServiceAccount -Identity "exchangeASA" -Properties servicePrincipalName | Select-Object Name,servicePrincipalName
+    ```
+
+    Restart Client Access Service webpool and Service
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md]:
```suggestion
    Restart Client Access Service webpool and Service.
```

File: Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md [hunk 11]
```diff
@@ -403,3 +403,76 @@ To configure your servers that are running Client Access services to stop using
...
+    ```Powershell
+    Get-ClientAccessService | Set-ClientAccessService -RemoveAlternateServiceAccountCredentials -CleanUpInvalidAlternateServiceAccountCredentials -Verbose
+    ```
+
+    Get CAS servers and add gMSA as ASA credential
+    ```Powershell
+    Get-ClientAccessService | Set-ClientAccessService -AlternateServiceAccountCredential $Credential -Verbose
+    ```
+
+    Verify ASA credential deployment
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md]:
```suggestion
    Verify ASA credential deployment.
```

File: Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md [hunk 12]
```diff
@@ -403,3 +403,76 @@ To configure your servers that are running Client Access services to stop using
...
+        Invoke-Command -ComputerName $CASExchangeHost -ScriptBlock {
+            Restart-WebAppPool -Name MSExchangeAutodiscoverAppPool 
+            Restart-Service MSExchangeServiceHost -Force
+        } 
+    }
+    ```
+
+
+# Create Credentials object with blank password
+Note that this is not meant to run as a single script and has no error checks.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md]:
```suggestion
Note that these are not meant to run as a single script and has no error checks.
```

File: Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md [hunk 13]
```diff
@@ -403,3 +403,76 @@ To configure your servers that are running Client Access services to stop using
...
+    Foreach ($CASExchangeHost in (Get-ClientAccessService).name){
+        Invoke-Command -ComputerName $CASExchangeHost -ScriptBlock {
+            Restart-WebAppPool -Name MSExchangeAutodiscoverAppPool 
+            Restart-Service MSExchangeServiceHost -Force
+        } 
+    }
+    ```
+
+
+# Create Credentials object with blank password
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer/architecture/client-access/kerberos-auth-for-load-balanced-client-access.md]:
```suggestion
# Create credentials object with blank password
```

**Comment by scanum** [comment]
@mattpennathe3rd PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request closed without merging**